doc/source/whatsnew/v1.0.0.rst

```diff
@@ -235,6 +235,71 @@ The following methods now also correctly output values for unobserved categories
    df.groupby(["cat_1", "cat_2"], observed=False)["value"].count()
 
 
+.. _whatsnew_1000.api_breaking.deps:
+
+Increased minimum versions for dependencies
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some minimum supported versions of dependencies were updated (:issue:`29766`).
+If installed, we now require:
+
++-----------------+-----------------+----------+
+| Package         | Minimum Version | Required |
++=================+=================+==========+
+| numpy           | 1.13.3          |    X     |
++-----------------+-----------------+----------+
```

Comment:
can u add a column that has an X if it changed in this version from previous (eg mark pyarrow)